dsl/generate_source_data.rb

```diff
@@ -105,11 +105,11 @@ def urls
 app_types << AppType.new('java', 'java-app-')
 app_types << AppType.new('flask', 'flask-app-')
 app_types << AppType.new('gov_uk', 'gov-uk-', gov_paths)
-app_types << AppType.new('dm-supplier-frontend', 'dm-supplier-frontend-app-', ['/suppliers'])
-app_types << AppType.new('dm-buyer-frontend', 'dm-buyer-frontend-app-', dm_buyer_paths)
-app_types << AppType.new('dm-admin-frontend', 'dm-admin-frontend-app-', ['/admin'])
-app_types << AppType.new('dm-search-api', 'dm-search-api-app-')
-app_types << AppType.new('dm-api', 'dm-api-app-')
+app_types << AppType.new('dm-supplier-frontend', 'dm-supplier-frontend-', ['/suppliers'])
```

Comment:
This change is required because the bulk_deploy script changed how it names apps.